src/prelude/common.mth

```diff
@@ -58,6 +58,11 @@ def(ptr+, Int Ptr -- Ptr, prim.unsafe.ptr+)
 def(ptr@, Ptr -- Ptr, @ Int->Ptr)
 def(ptr!, Ptr Ptr --, dip(Ptr->Int) !)
 
+def(ptr, Size, 8)
```

Comment:
I'm not happy about the magic 8 here :shrug:

Reply:
I think you need either a "pointer size" (or some way to get pointer size, like an "architecture" primitive) to deal with the x86 target.

Reply:
indeed! I was surprised it hadn't been necessary so far!